amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/config/KinesisClientLibConfigurator.java

```diff
@@ -110,4 +111,12 @@ public MultiLangDaemonConfiguration getConfiguration(InputStream configStream) {
         }
         return getConfiguration(properties);
     }
+
+    private String processKey(Map.Entry<Object, Object> e) {
+        String key = (String) e.getKey();
```

Comment:
do we have a test for this?

Reply:
Changed to default access modifier to allow testing. Added a few tests for expected keys.